spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/util/FormUtils.java

```diff
@@ -997,7 +1022,7 @@ private static void processDilutions(String dilutionMolarity, Library library, P
           log.info("Added library dilution: " + ldi.toString());
         }
         if (p != null) {
-          p.getPoolableElementViews().add(PoolableElementView.fromDilution(ldi));
+          p.getPoolDilutions().add(new PoolDilution(p, PoolableElementView.fromDilution(ldi)));
           log.info("Added library dilution to pool: " + p.toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)